curvefs/util/install.sh

```diff
@@ -0,0 +1,232 @@
+#!/usr/bin/env bash
+
+# Copyright (C) 2021 Jingli Chen (Wine93), NetEase Inc.
+
+# metaserevr
+# ├── conf
+# │   └── metaserver.conf
+# ├── logs
+# │   └── metaserevr.pid
+# ├── sbin
+# │   ├── servicectl
+# │   └── curvefs-metaserver
+# └── lib
```

Comment:
> `lib` is unnecessary

I think the `lib` is necessary, because our binary depends on some dynamic libraray, like `jemalloc.so`、 `libetcdclient.so`, i don't want install these library to the OS system path (maybe `/usr/lib`), it will pollution system environment.